src/module.js

```diff
@@ -1265,7 +1308,8 @@ export default class DailyIframe extends EventEmitter {
         if (!p.audioTrack) { p.audio = false };
```

Comment:
if we stop subscribing to a track by invoking `updateParticipants("foo", { audio: false })` when we're processing the participant update event here will `p.audio` properly be set to `false` so that we can emit the `track-stopped` event?

I'll probably find out soon when I review the `pluot-core` side of things, but just wanted to make a note here to check

Reply:
actually, what *should* we see in a particular `participant` object returned by `callObject.participants()` if that participant *does* have an audio track but we're not subscribed to it? should it be:

```js
{
  // ...
  audio: true,
  // ...
}
```

and no `audioTrack` field? Should there be something in there indicating that we're intentionally omitting the `audioTrack`, for ease of customer code consuming this object?

Reply:
Unsubscribing from a track generates a `track-stopped` event.

Changing the mode to not subscribe to tracks automatically generates `track-stopped` events for all tracks.

Reply:
Great question about what we should actually be doing for a track that is being sent, but that you are not subscribed to. This seems similar to the issue that come up when building the React demo: we currently don't have any way to build a "this person is sending a track but it's not playable for you, yet" UI, with only information available at the API level.

That would be a great thing to fix. Here's a proposal for a change to the participant structure:

```
{
  audio: true,   // the remote participant is sending audio, or ...
  audio: false,  // the remote participant is muted or has no mic
  receiving: {
    audio: {
      playable: true,  // we have live media. in call object mode we'll also have a track if this is true
      subscribed: true,
      track: null | <track>,  // only in call object mode
      bandwidthCap: <kbs>, // future receive-side bandwidth control feature
    },
    ... // same thing for video and screenVideo
  }

Reply:
I like the overall structure. I like that it is expandable if/when we want to send more metadata. That said, I think `receiving` is misleading since I would think it would only be filled with tracks that I'm actually receiving (subscribed to). `tracks` makes more sense to me but i'm not as well versed in the lingo.

Reply:
`tracks` is a great idea.

Reply:
I like this proposal! And the `tracks` nomenclature. Even though `tracks.audio.track` is a bit odd to read, it's not unclear.

Reply:
Currently `audioTrack` would just be missing if the recipient is not subscribed to it, right?